Quantum-Control-Applications/Superconducting/Single Fixed Transmon/Use Case 3 - DRAG calibration/README.md

```diff
@@ -0,0 +1,75 @@
+# Derivative Removal by Adiabatic Gate (DRAG) calibration
+
+_Author: Kevin Villegas Rosales_
+
+_Important note: The code in this folder is the exact code that we used
+to run the experiment, so it is tailored for the specific software environment
+and set up. Thus, this is code is for inspirational purposes._
+
+## The goal
+
+The goal of this experiment is to calibrate the DRAG coefficient
+to increase the single qubit gate fidelities as measured with 
+randomized benchmarking (RB).
+
+## The device
+
+The chip contains 5 2D-transoms, where some of them have nearest-neighbor connectivity, 
+with flux tunable lines. For this experiment, we focus only in one transmon with a drive
+to its flux line to put the transmon  in the **sweep-spot**. All other transmons are left
+unbiased and signal is send to them.
+
+## Method and results
+
+We will show the determination of the DRAG coefficient with two different pulse sequences.
+The first one is related to the application of `(x180)(-x180)` train of pulses and the second
+is a sequence of `x180y90` and `y180x90` pulses. We are calibrating the `$\alpha$` parameter in 
+the following formula,
+
+<img src="https://latex.codecogs.com/svg.image?\Omega^{'} = \Omega + i\alpha\frac{d\Omega/dt}{\Delta}" />
+
+### DRAG coefficient calibration with `x180` `-x180` pulses
+
+In this step we generate a 2D plot where the axis are `number of (x180)(-x180)` pulses
+that applied, and the other axis is the `DRAG coefficient`. We gain sensitivitiy 
```

Comment:
Applying more pulses increases the accumulation of population in the 2nd excited state (leakage), allowing us to tune the alpha parameter more accurately.
OR
Applying more pulses increases the leakage to the 2nd excited state, allowing us to tune the alpha parameter more accurately.

Reply:
I rephrased it